hordelib/model_manager/lora.py

```diff
@@ -44,14 +44,19 @@ class LoraModelManager(BaseModelManager):
     )
     LORA_API = "https://civitai.com/api/v1/models?types=LORA&sort=Highest%20Rated&primaryFileOnly=true"
     MAX_RETRIES = 10 if not TESTS_ONGOING else 3
-    MAX_DOWNLOAD_THREADS = 3
+    MAX_DOWNLOAD_THREADS = 5 if not TESTS_ONGOING else 15

```

Comment:
I'm surprised the number of download threads makes a big difference. not only are they mostly relevant when one does not have default loras, but I would even if you download more than 3 at a time, I would assume you would see you max out your download speed